src/protobuf-net.Test/Nullables/CollectionsWithNullsTests_Legacy.cs

```diff
@@ -0,0 +1,34 @@
+﻿using ProtoBuf.Test.Nullables.Abstractions;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization;
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ProtoBuf.Test.Nullables
+{
+    public class CollectionsWithNullsTests_Legacy : CollectionsWithNullsTestsBase
+    {
+        public CollectionsWithNullsTests_Legacy(ITestOutputHelper log) 
+            : base(log)
+        {
+        }
+
+        [Fact]
+        public void ExistingListsBehaviour()
+        {
+            using var ms = new MemoryStream(new byte[] { 0x08, 0x00, 0x08, 0x01, 0x08, 0x02 });
+            var clone = Serializer.Deserialize<Foo>(ms);
+            if (!ms.TryGetBuffer(out var buffer)) buffer = new ArraySegment<byte>(ms.ToArray());
+            Assert.Equal("08-00-08-01-08-02", BitConverter.ToString(buffer.Array, buffer.Offset, buffer.Count));
```

Comment:
we didn't serialize, so what else do we expect? note: if this was in the code you copied from me: blame me!

Reply:
removed legacy test completely - we have `CollectionsWithNullsTests_Basic`, which checks schema \ payload \ serialization behavior